Doppler.AccountPlans/Infrastructure/AccountPlansRepository.cs

```diff
@@ -29,5 +29,24 @@ public async Task<IEnumerable<PlanDiscountInformation>> GetPlanDiscountInformati
 
             return result;
         }
+
+        public async Task<IEnumerable<PlanInformation>> GetPlanInformation(int planId)
```

Comment:
I think that this method should be return only PlanInformation and not a `IEnumerable`, right?